mne/viz/_3d.py

```diff
@@ -1344,3 +1345,135 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def plot_dipole_3d(dipole, trans, subject, subjects_dir=None, scale_factor=1e9,
```

Comment:
Is that safe? Does someone really want to plot the dipoles without applying the transformation?